app/src/renderer/components/transactions/LiAnyTransaction.vue

```diff
@@ -88,6 +93,12 @@ export default {
     type() {
       return this.transaction.tx.value.msg[0].type
     },
+    fees() {
+      return (
+        this.transaction.tx.value.fee.amount &&
```

Comment:
May happen that `this.transaction.tx.value.fee` is `undefined`?

Reply:
yes, if you send a tx without fees it returns `null`

Reply:
He meant `this.transaction.tx.value.fee`